extension/content.js

```diff
@@ -99,10 +99,26 @@ function renderVoteCount(type, count) {
 		iconUrl = 'https://assets-cdn.github.com/images/icons/emoji/unicode/1f44e.png';
 	}
 	const $sidebar = $('#partial-discussion-sidebar');
+	let avatars = '';
+
+	voters.forEach(val => {
+		avatars += val;
+	});
+
+	avatars = avatars.replace(/height="48"/g, 'height="20"')
+										.replace(/width="48"/g, 'width="20"')
+										.replace(/<a href/g, '<a class="participant-avatar" href')
+										.replace(/timeline-comment-avatar/g, 'avatar');
```

Comment:
IDK exactly why, but adding DOM nodes to the `Set` is not playing nice. Either I'm getting `undefined` by the time it gets to inserting the element, or it's not counting the votes correctly when they're being added to the `Set`.

Is DOM manipulation going to be that much better?